### ISSUE TITLE
Devbox /fhir API  returns a different fhir structure than expected

### ISSUE THREAD

**Issue by Christian9312** [Aidbox/Issues#19]
When using Devbox /fhir API to create a `QuestionnaireResponse` resource, the returned resource has a different structure to expected.  The `ValueCoding` in response is splited in a `Coding` record inside a `Value` record.

Devbox Version: -15128758-0
Ex: Post `http://localhost:8888/fhir/QuestionnaireResponse`
```json
{
    "resourceType": "QuestionnaireResponse",
    "identifier": {
        "system": "http://example.com",
        "value": "1375da80-1cb1-11e8-902b-c77f228d89bb"
    },
    "status": "completed",
    "subject": {
        "display": "testuser"
    },
    "item": [
        {
            "linkId": "Biometrics",
            "item": [
                {
                    "linkId": "ht",
                    "text": "What is your height in feet.inches?",
                    "answer": [
                        {
                            "valueCoding": {
                                "system": "http://example.com",
                                "code": "6"
                            }
                        }
                    ]
                }
            ]
        }
    ]
}
```
Response 201 `Created` :
```json
{
    "item": [
        {
            "item": [
                {
                    "text": "What is your height in feet.inches?",
                    "answer": [
                        {
                            "value": {
                                "Coding": {
                                    "code": "6",
                                    "system": "http://example.com"
                                }
                            }
                        }
                    ],
                    "linkId": "ht"
                }
            ],
            "linkId": "Biometrics"
        }
    ],
    "status": "completed",
    "identifier": {
        "value": "1375da80-1cb1-11e8-902b-c77f228d89bb",
        "system": "http://example.com"
    },
    "id": "bdbe73cd-6544-4227-9029-4a631a27efbc",
    "meta": {
        "lastUpdated": "2018-11-19T20:45:37.713Z",
        "versionId": "41",
        "tag": [
            {
                "system": "https://aidbox.app",
                "code": "created"
            }
        ]
    },
    "subject": {
        "display": "testuser"
    },
    "resourceType": "QuestionnaireResponse"
}
```

**Comment by mlapshin** [comment]
Thanks for reporting this. We'll fix it ASAP.

**Comment by mlapshin** [comment]
fixed in `healthsamurai/devbox:0.3.3-73-ge64f17d`
